### PR TITLE
Makes sure that new generated components are ordered alphabetically

### PR DESCRIPTION
**Pull request by EgoziE**
.wuf/components.json is safe to remove

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "storybook:build": "npm run prestorybook && build-storybook -s ./src/assets",
     "------new components generators": "",
     "generate": "wuf generate --output src --codemods .wuf/generator/codemods --templates .wuf/generator/templates",
-    "postgenerate": "npm run update-components",
+    "postgenerate": "rm ./.wuf/components.json && npm run update-components",
```
Comment (argshook) [inline package.json:32]: should be part of `.wuf/update-components.sh`

Comment (EgoziE) [reply package.json:32]: good idea, i'll change it

Comment (EgoziE) [reply package.json]: changed and verified it works

File: .wuf/update-components.sh [hunk 2]
```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+rm ./.wuf/components.json
```
Comment (argshook) [inline .wuf/update-components.sh:3]: sorry for nitpicking a bit more :)

```suggestion
rm .wuf/components.json || true
```

1. the `./` is not necessary at the beginning of path
2. the `|| true` part will allow `rm` to fail but continue executing the rest of the script.
   that's important in the case if `.wuf/components.json` doesn't exists for example, freshly cloned repo.
   `rm` would throw and the rest of `update-components.sh` would be prevented from executing

Comment (EgoziE) [reply .wuf/update-components.sh:3]: cool thanks

**Pull request merged**